### DISCUSSION TITLE
建议专心搞润学

### DISCUSSION THREAD

**Discussion by godjoem** [The-Run-Philosophy-Organization/run#169] [General]
润学repo放关于润的方法论就可以了，来这儿的人都是目的明确的，搞这个人那个人的言论挂出来对润毫无帮助让润不纯粹

**Comment by ikingye** [comment]
认真看了这一期，感觉楼主是反串啊，西方所谓言论自由，确实就是双标嘛。

https://www.bilibili.com/bangumi/play/ep301215

<img width="826" alt="image" src="https://user-images.githubusercontent.com/29562589/163713557-5c8c299b-a3b9-4631-b5af-16a04f6c7913.png">
